### PR TITLE
Work started on migrations and relationships

### PR DESCRIPTION
**Pull request by annadolan**
@jmejia @meganft Here is the work I've been doing on our schema. I'm adding relationships as well, please look it over and make sure I am not missing any belongs_to or has_many on the models. I'll be adding model tests on this branch as well. Thanks!

Here's the schema I'm working with: 
screen shot 2017-01-05 at 9 53 58 am

### CODE REVIEW

File: app/models/category.rb [hunk 1]
```diff
@@ -1,3 +1,5 @@
 class Category < ApplicationRecord
   has_many :tasks
+
+  validates :name, presence: true
 end
```
Comment (jmejia) [inline app/models/category.rb:5]: Any chance we'd want a task to have multiple categories? If so, we could create a has_many through.

Comment (jmejia) [reply app/models/category.rb:5]: @annadolan ^^

Comment (annadolan) [reply app/models/category.rb:5]: @jmejia that's a good point. I think we might hold off based on Rachel's slides, but we will bring that up when we talk to her today. Thanks!

**Comment by annadolan** [comment]
@meganft Should be ready to merge if you're happy with everything! All tests are passing. 🍪

**Comment by meganft** [comment]
Great work @annadolan 
💯 ❄️

**Pull request merged**